File: pinyougou_parent/pinyougou_cart_interface/src/main/java/com/pinyougou/cart/service/CartService.java
package com.pinyougou.cart.service;

import groupEntity.Cart;

import java.util.List;

public interface CartService {
    public List<Cart> addItemToCartList(List<Cart> cartList,Long itemId,Integer num);

    List<Cart> selectCartListFromRedis(String sessionId);

    void saveCartListToRedis(String sessionId, List<Cart> cartList);

    List<Cart> mergeCartList(List<Cart> cartList_sessionId, List<Cart> cartList_userName);

    void deleteCartList(String sessionId);
}

File: pinyougou_parent/pinyougou_order_web/src/main/webapp/js/service/cartService.js
app.service('cartService',function($http){

    //购物车列表数据
    this.findCartList=function(){
        return $http.get('cart/findCartList.do');
    }

    this.addItemToCartList=function (itemId,num) {
        return $http.get('cart/addItemToCartList.do?itemId='+itemId+"&num="+num);
    }

});
